packages/server/src/service/pdfForm/westVirginia.ts

```diff
@@ -0,0 +1,140 @@
+import { WestVirginiaInfo } from '../../common'
+import { fillFormWrapper  } from '.'
+import { cleanPhoneNumber, toSignatureBuffer } from './util'
+
+export const fillWestVirginia = (
+  stateInfo: WestVirginiaInfo
+) => fillFormWrapper(
+  'West_Virginia.pdf',
+  async ({check, text, placeImage}) => {
+
+    const { nameParts } = stateInfo
+
+    // First name.
+    text(nameParts.first, 0, 277, 117)
+    // Middle initial.
+    if (nameParts.middle) {
+      text(nameParts.middle, 0, 432, 117)
+    }
+    // Last name.
+    text(nameParts.last, 0, 132, 117)
+    // Suffix.
+    if (nameParts.suffix) {
+      text(nameParts.suffix, 0, 572, 117)
+    }
+
+    // Birth year.
+    const birthMonth = stateInfo.birthdate.split('/')[0]
+    const birthDay = stateInfo.birthdate.split('/')[1]
+    const birthYear = stateInfo.birthdate.split('/')[2]
+    text(birthMonth[0], 0, 480, 154)
```

Comment:
```
text(stateInfo.birthdate.split('/')[0], 0, 480, 154)
text(stateInfo.birthdate.split('/')[1], 0, 515, 154)
text(stateInfo.birthdate.split('/')[2], 0, 557, 154)
```

Reply:
Nope, I assumed that was the practice you guys followed based on Virginia, but I'll change it to be grouped.